plugin/dirvish.vim

```diff
@@ -3,6 +3,8 @@ if exists('g:loaded_dirvish') || &cp || v:version < 700 || &cpo =~# 'C'
 endif
 let g:loaded_dirvish = 1
 
+let g:dirvish_shdo_preamble = get(g:, 'dirvish_shdo_preamble', '')
```

Comment:
"before" or just "pre" is more common/familiar/conventional, no?
```suggestion
let g:dirvish_shdo_before = get(g:, 'dirvish_shdo_preamble', '')
```

Reply:
Sure! Before sounds good to me.